vocab/credentials/v2/vocabulary.yml

```diff
@@ -23,17 +23,21 @@ class:
 
   - id: CredentialSchema
     label: Credential schema
-    comment: A Credential Schema provides verifiers with enough information to determine if the provided data conforms to the provided schema. This class serves as a supertype for specific schemas (e.g., <a href="#JsonSchema2023">JsonSchema2023</a>).
+    comment: A Credential Schema provides verifiers with enough information to determine if the provided data conforms to the provided schema. This class serves as a supertype for specific schemas (e.g., <a href="#JsonSchema">JsonSchema</a>).
 
   - id: CredentialStatus
     label: Credential status
     comment: A Credential Status provides enough information to determine the current status of the credential (for example, suspended or revoked). This class serves as a supertype for specific status types.
 
-  - id: JsonSchema2023
-    label: JSON schema validator 2023
-    defined_by: https://www.w3.org/TR/vc-json-schema/#jsonschema2023
+  - id: JsonSchema
+    label: JSON schema validator
+    defined_by: https://www.w3.org/TR/vc-json-schema/#jsonschema
+    upper_value: cred:CredentialSchema
+
+  - id: JsonSchemaCredential
+    label: JSON schema validator with schema contained in a credential
+    defined_by: https://www.w3.org/TR/vc-json-schema/#jsonschemacredential
     upper_value: cred:CredentialSchema
```

Comment:
@iherman nitpick here... I think the upper value of `JsonSchemaCredential` is a `VerifiableCredential` ... not a CredentialSchema.

Reply:
Not nitpick, but a bug :-)

I have updated it accordingly.